Cargo.toml

```diff
@@ -54,3 +60,9 @@ cosmos = ["secp256k1"]
 erc20 = ["secp256k1", "web3"]
 ethereum = ["secp256k1", "web3"]
 solana = ["ed25519-dalek"]
+build-binary = ["clap"]
+
+[[bin]]
+name = "cli"
+path = "src/client/bin/cli.rs"
+required-features = ["build-binary"]
```

Comment:
Not a big thing, but there's a new line missing at the end of the file. Maybe this project should have editorconfig in the root, most of the editors already are aware of it.